lib/flight_job/models/question.rb

```diff
@@ -86,7 +86,8 @@ def validate_schema
     end
 
     def default
-      return @default if @dynamic_default.nil?
+      @default ||= nil
```

Comment:
If `@default` is set to `false`, this line will change it to `nil`.  Would the following work (this is totally untested)

```ruby
def default
  if defined?(@dynamic_default)
    generate(**@dynamic_default) || @default
  else
    @default
  end
end
```

Reply:
I added this line as the spec was showing a warning saying that `@default` hadn't been initialised. I hadn't realised that `||=` would overwrite the variable if it had been defined as false - good to know! So a line making sure that `@default` has been defined is needed - I've had a go at changing it below...